extensions/container-image/container-image-openshift/deployment/src/main/java/io/quarkus/container/image/openshift/deployment/S2iConfig.java

```diff
@@ -23,15 +20,6 @@ public class S2iConfig {
     public static final String FALLBACK_JAR_DIRECTORY = "/deployments/";
     public static final String FALLBACK_NATIVE_BINARY_DIRECTORY = "/home/quarkus/";
 
-    public static String getDefaultJvmImage(CompiledJavaVersionBuildItem.JavaVersion version) {
-        switch (version.isJava21OrHigher()) {
-            case TRUE:
-                return DEFAULT_BASE_JVM_JDK21_IMAGE;
-            default:
-                return DEFAULT_BASE_JVM_JDK17_IMAGE;
-        }
-    }
```

Comment:
You removed this. Is it because it's not used? Because it's not doing the same thing as the one in `ContainerImages`.

Reply:
No it was not used.